spec/requests/portfolios_spec.rb

```diff
@@ -52,6 +52,24 @@
       end
     end
 
+    describe "POST admin tagged /portfolios/:portfolio_id/portfolio_items" do
+      let(:portfolio_without_item) { create(:portfolio) }
+      let(:portfolio_item_to_add) { create(:portfolio_item) }
+      let(:params) { {:portfolio_item_id => portfolio_item_to_add.id} }
+      before do
+        post "#{api}/portfolios/#{portfolio_without_item.id}/portfolio_items", :params => params, :headers => send("#{tag}_headers")
+      end
+
+      it 'returns a 200' do
+        expect(response).to have_http_status(200)
+      end
+
+      it 'returns the portfolio_item we just added to the portfolio' do
+        expect(json.size).to eq 1
+        expect(json.first['portfolio_id']).to eq portfolio_without_item.id
```

Comment:
This `expect` is confusing with the `portfolio_without_item` var name. I think the `it` statement does a good job of describing what is happening.

Reply:
Got them updated!